library/common/config/config.cc

```diff
@@ -39,6 +39,9 @@ const std::string config_header = R"(
 - &dns_fail_max_interval 10s
 - &dns_query_timeout 25s
 - &dns_preresolve_hostnames []
+- &h2_connection_keepalive_interval 0s
+- &h2_connection_keepalive_idle_interval 0s
+- &h2_connection_keepalive_timeout 5s
```

Comment:
It looks like these still need to be referenced in the cluster config.

Reply:
Yep, I didn't want to finish wiring until we settled on interface.